src/sbt-test/sbt-accessibility-linter/e2e-both-linters-found-no-problems/test

```diff
@@ -0,0 +1,3 @@
+# checks that you can use the accessibility 
+> a11yInstall
+> test
```

Comment:
This works, but I'm not sure how to add an assertion on the contents of stdout 🤔